userguide/tutorials/using-ses-with-aws.adoc

```diff
@@ -0,0 +1,51 @@
+ 
+ 1. Sign in to AWS.
+ 2. Go to the https://console.aws.amazon.com/sesv2/[SMTP dashboard]. You will see information about the SMTP server you are creating, including the fact that your server will be in the sandbox for your region. A sandbox server is easier to setup and suitable for development, but it places limits on how you can use the server. You can exit the sandbox later.
+ 3. Scroll to the bottom and click on *Create SMTP Credentials*.
+ 4. A brief form will appear. Accept the username and click *Create* at the bottom of the page.
+ 5. You will see a message that your user has been created successfully. Click on *show user SMTP security credentials* to display the generated username and password. These must be saved, as they will be required to be copied to the plugin configuration in a later step.
+ 6. Click *Download Credentials* to save your credentials in a comma-separated text file.
+ 
+### Verifying email Identities
+
+SES requires that any email address to be used as the MAIL FROM or REPLY TO address in your messages should be verified. This is similar to the identity verification process performed by an SSL certificate. While you are in the sandbox, all destination email addresses must be verified as well.
+
+It is possible to verify individual email addresses or complete domains. We recommend that you verify the domain that is used by the individual email addresses you will be using. This requires setting up several CNAMEs in the DNS settings of a domain that you own. If you would prefer to verify individual emails, which avoids this need, see https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html#verify-email-addresses-procedure[Verifying an email Address].
+
+A CNAME is an identifier that links a resource to be identified to your own domain. Your domain provider should have instructions for setting this up.
+
+General instructions for verifying a domain are given in https://docs.aws.amazon.com/ses/latest/dg/creating-identities.html#verify-domain-procedure[creating and verifying a domain identity]. You can follow these instructions in the simplest way:
+
+- Do not assign a default configuration
+- Do not use a custom MAIL FROM address
+- Accept the default Domain Keys Identified Mail (DKIM) tokwn
+- No tags are required
+
+The authentication tab will provide you with three CNAME records. Each of these must be added to your domain's DNS settings.
+
+WHen these changes are detected by SES, the identity status for your domain will appear as *verified*. this can take up to 48 hours although often it will take a lot less.
+
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/SES-verified-identities.jpg[align=center]
+
+
+### Opening Port 25
+
+SMTP uses Port 25 to send mail. By default this port is disabled. To begin, add an entry to your security group of type SMTP. This type will use port 25 by default. Set the Source to 0.0.0.0/0 as the others are set, and save the new rule.
```

Comment:
Slightly confused about this: The `email-notifications-plugin` will use port 25 to reach the SMTP server -- but it will not **receive** anything on port 25, so why do we need to open the (incoming) port?

Reply:
You are correct. This is not necessary.